recipes/libnl-devel-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: libnl-devel-cos6-x86_64
+  version: 1.1.4
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libnl-devel-1.1.4-2.el6.x86_64.rpm
+    sha256: 6c1749d9f8e9cbe31c84067a7da5f9861c2cd61a0df9b83107b6d045f24a766a
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/libnl-1.1.4-2.el6.src.rpm
+    folder: source
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - libnl-cos6-x86_64 ==1.1.4
+  run:
+    - libnl-cos6-x86_64 ==1.1.4
+
+###########################################################
+# Commented out to avoid a conda-build bug.               #
+# xref: https://github.com/conda/conda-build/issues/3553  #
+###########################################################
+#
+#test:
+#  commands:
+#    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
```

Comment:
Currently adding a `test` section results in issue ( https://github.com/conda/conda-build/issues/3553 ). So it is commented for now.

Reply:
Have added a workaround to get the tests back here.